kostyor/db/migrations/alembic/versions/2bedb47d68e6_initial_schema.py

```diff
@@ -0,0 +1,81 @@
+"""Initial database schema
+
+Revision ID: 2bedb47d68e6
+Revises: initial
+Create Date: 2016-09-09 11:02:40.083013
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '2bedb47d68e6'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+STATUSES = ['READY FOR UPGRADE', 'UPGRADE IN PROGRESS', 'UPGRADE PAUSED',
```

Comment:
I think fixing constants should be done in separate patch, not in this one. We use space-based constants in the code anyway.